server/src/main/java/org/opensearch/action/support/broadcast/node/TransportBroadcastByNodeAction.java

```diff
@@ -532,6 +532,13 @@ private void onShardOperation(
         }
     }
 
+    /**
+     * This method reads ShardRouting from input stream
+     */
+    public List<ShardRouting> getShardsFromInputStream(StreamInput in) throws IOException {
```

Comment:
It looks like `getShardRoutingsFromInputStream` would better reflect what this method is doing